src/components/WebexActivityStream/Greeting.scss

```diff
@@ -8,8 +8,21 @@ $C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-greeting;
   max-width: 37.5rem;
   text-align: center;
   color: var(--wxc-text-color);
+  margin: 0 auto;
+  margin-top: 11.25rem;
+
+  &--mobile {
```

Comment:
We should have a mobile-first approach for the styling, considering that most cases the widgets will be used in small spaces.

This means that default styles should be for small/mobile screens and then overwritten for larger sizes

Reply:
I ended up doing the opposite, because for some reason it would only recognize the mobile media query